pkg/pillar/cmd/zedkube/applogs.go

```diff
@@ -50,11 +56,14 @@ func collectAppLogs(ctx *zedkubeContext) {
 	sinceSec = logcollectInterval
 	for _, item := range items {
 		aiconfig := item.(types.AppInstanceConfig)
+		if aiconfig.FixedResources.VirtualizationMode != types.NOHYPER {
+			continue
+		}
 		if aiconfig.DesignatedNodeID != uuid.Nil && aiconfig.DesignatedNodeID.String() != ctx.nodeuuid { // For now, only check DNiD, need to add migration part
 			continue
 		}
-		contName := base.GetAppKubeName(aiconfig.DisplayName, aiconfig.UUIDandVersion.UUID)
-
+		kubeName := base.GetAppKubeName(aiconfig.DisplayName, aiconfig.UUIDandVersion.UUID)
+		contName := kubeName
```

Comment:
Do we need both here?

Reply:
i think this is the case, the contName can change, the kubeName can be the prefix of a real name:
```
		for _, pod := range pods.Items {
			if strings.HasPrefix(pod.ObjectMeta.Name, kubeName) {
				contName = pod.ObjectMeta.Name
				break
			}
		}
```